src/site/pages/upgrade-guide.md

```diff
@@ -0,0 +1,223 @@
+    .pipe(sassGlob())
+    .pipe(replace('@import "./@node_modules/@patternfly/patternfly-next/patternfly-utilities";', ''))
+    .pipe(gulp.dest('./css'))
+);
+```
+
+- Import all specific modules
+
+```js
+gulp.task('compile-scss', () =>
+  gulp
+    .src('./@node_modules/@patternfly/patternfly-next/**/*.scss')
+    .pipe(insert.append('@import "./@node_modules/@patternfly/patternfly-next/layouts/Page/*.scss";\n'))
+    .pipe(insert.append('@import "./@node_modules/@patternfly/patternfly-next/layouts/Grid/*.scss";\n'))
+    .pipe(insert.append('@import "./@node_modules/@patternfly/patternfly-next/components/Content/*.scss";\n'))
+    .pipe(insert.append('@import "./@node_modules/@patternfly/patternfly-next/components/Content/*.scss";\n'))
+    .pipe(sassGlob())
+    .pipe(replace('@import "./@node_modules/@patternfly/patternfly-next/patternfly-utilities";', ''))
+    .pipe(gulp.dest('./css'))
+);
+```
+
+#### Webpack
+
+_This example uses the following configuration:_
+
+```json
+webpack: "3.8.1",
+sass-loader: "7.0.0",
+css-loader: "^0.28.11",
+style-loader: "^0.21.0",
+node-sass: "^4.9.0"
+```
+
+_Code Snippets_
+
+- Import All `.scss` files
+
+```js
+module.exports = {
+  module: {
+    rules: [{
+      test: /\.scss$/,
+      use: [{
+        loader: "style-loader"
+      }, {
+        loader: "css-loader"
+      }, {
+        loader: "sass-loader",
+        options: {
+          includePaths: [
+            "../node_modules/@patternfly/patternfly-next/"
+          ]
+        }
+      }]
+    }]
+  }
+};
+```
+
+- Import select modules
+
+```js
+module.exports = {
+  module: {
+    rules: [{
+      test: /\.scss$/,
+      use: [{
+        loader: "style-loader"
+      }, {
+        loader: "css-loader"
+      }, {
+        loader: "sass-loader",
+        options: {
+          includePaths: [
+            "../node_modules/@patternfly/patternfly-next/layouts/Page/",
+            "../node_modules/@patternfly/patternfly-next/layouts/Grid",
+            "../node_modules/@patternfly/patternfly-next/components/Content",
+            "../node_modules/@patternfly/patternfly-next/components/SecondaryNav",
+            "../node_modules/@patternfly/patternfly-next/components/Button"
+          ]
+        }
+      }]
+    }]
+  }
+};
+```
+
+----
+
+## Typography
+
+With PatternFly 4, we are switching from the Open Sans font family to Overpass. Additionally, we are updating the base font size from `12px` to `16px`, in order to increase readability and accessibility.
+
+Overpass can be utilized in two different ways:
+
+1. Built into PatternFly 4.
+
+- By default, we include Overpass as part of the PatternFly 4 distributed CSS file. You do not need to do anything with your configuration to use this new font family.
+
+2. Use as a CDN
+
+- If you wish to use the CDN for Overpass, rather than the default approach, you will need to update the `utilities/variables.scss` file and build PatternFly-Next as part of your build process.
```

Comment:
You can lose the comma before "rather than" and just keep the comma following "approach"

Reply:
Also, PF-Next should be PF 4 here